plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/StandardDeployPreferencesPanel.java

```diff
@@ -161,6 +161,11 @@ public IStatus validate(Object value /* email */) {
         .setConverter(new Converter(String.class, String.class) {
       @Override
       public Object convert(Object fromObject /* email */) {
+        // If there exists only one account signed in, make it selected.
+        String singleAccountEmail = accountSelector.getSingleAccountEmail();
```

Comment:
I'm not sure but I'm tempted to go a third way. It looks cleaner to me (though I'm not 100% sure of this) to not add anything to AccountSelector, check if there's only one, and if so set it here. But I don't have all the code in front of me, so maybe not.